domain/src/main/kotlin/no/nav/su/se/bakover/domain/satser/SatsFactoryForSupplerendeStønad.kt

```diff
@@ -90,12 +118,20 @@ class SatsFactoryForSupplerendeStønad(
         return cache.getOrAdd(grunnbeløpsendringer.gjeldende(påDato).ikrafttredelse)
```

Comment:
Her må vi gjøre noe tilsvarende for garantipensjon slik at satsfactoryet man får tilbake også justerer garantipensjon i henhold til `påDato`.